packages/core/src/config.ts

```diff
@@ -247,6 +247,41 @@ export const composeAutoExternalConfig = (options: {
     : {};
 };
 
+export function composeMinifyConfig(
+  minify: NonNullable<RsbuildConfig['output']>['minify'],
+): RsbuildConfig {
+  if (minify !== undefined) {
+    // User's minify configuration will be merged afterwards.
+    return {};
+  }
+
+  // When minify is not specified, Rslib will use a sane default for minify options.
+  // The default options will only perform dead code elimination and unused code elimination.
+  return {
+    output: {
+      minify: {
+        js: true,
```

Comment:
Why do need minify JS files by default

Reply:
Libraries usually ship uon-minified code to npm.

Reply:
By default, it's not minified. Adding `minify: true` is required as we want to perform some tree-shaking related compress options by default (https://github.com/web-infra-dev/rslib/blob/d68915dd2294a0a0d2e767e9ed672c25fdb2f6a8/packages/core/src/config.ts#L288-L294).

Reply:
get